scripts/upgrade

```diff
@@ -246,7 +246,7 @@ then
     exec_occ -V
 
     if [ "$(exec_occ config:system:get mysql.utf8mb4)" != "true" ]; then
-        exec_occ db:convert-mysql-charset
+        exec_occ maintenance:repair
```

Comment:
Shouldn't you set also the `mysql.utf8mb4` attribute to true...?

Reply:
True...  I guess we also need to alter the database character set... (I am not able to test this on my setup for now...)